src/styles/layout/sidebar.less

```diff
@@ -903,6 +903,23 @@ Sidebar Open
 
   }
 
+  @media (max-height: @screen-medium-height) {
+
+    .sidebar {
+
+      &.is-expanded {
+
+        .sidebar-content {
+
+          .sidebar-logo-container {
+            display: none;
+          }
+        }
+      }
+    }
```

Comment:
spacing is weird here. these closing braces don't have newlines between them but the opening ones do